pages/article/ArticleHandler.inc.php

```diff
@@ -17,18 +17,47 @@
 import('classes.handler.Handler');
 
 class ArticleHandler extends Handler {
-	/** journal associated with the request **/
+	
+	/** @var journal object: journal associated with the request **/
 	var $journal;
 
-	/** issue associated with the request **/
+	/** @var issue object: issue associated with the request **/
 	var $issue;
 
-	/** article associated with the request **/
+	/** @var article object: article associated with the request **/
 	var $article;
 
-	/** galley associated with the request **/
+	/** @var galley object: galley associated with the request **/
 	var $galley;
-
+	
+	/** @var int: type of the requested object (article, galley or file) **/
+	var $type;
+	
+	/** @var int: article ID associated with the request **/
+	var $articleId;
```

Comment:
Hi @bozana. I took over from @felixgruender and removed the $articleId and uses $this->article->getId(). Thanks for the tip! The current version of the code can be viewed here: https://github.com/lilients/ojs and here https://github.com/lilients/pkp-lib. It is not yet final though.